images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM oryxdevmcr.azurecr.io/private/oryx/githubrunners-buildpackdeps-${DEBIAN_FLA
 ARG DEBIAN_FLAVOR
 ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR
 # Install basic build tools
-RUN LANG="C.UTF-8" \
+RUN export LANG="C.UTF-8" \
```

Comment:
When setting environment variables that we want to persist to the container that the image is ran against, we should use `ENV` in the Dockerfile rather than `RUN export` as the latter only persists for the single command call.

[This post](https://stackoverflow.com/a/33379487) should provide a little more information.

Reply:
Thanks for the comment! I've just changed the RUN to ENV variables. It's currently in validation pipeline.